### PR TITLE
[Merged by Bors] - Blinded block and RANDAO APIs

### PR DESCRIPTION
**Pull request by michaelsproul**
## Issue Addressed

https://github.com/ethereum/beacon-APIs/pull/241
https://github.com/ethereum/beacon-APIs/pull/242

## Proposed Changes

Implement two new endpoints for fetching blinded blocks and RANDAO mixes.

### CODE REVIEW

**Comment by michaelsproul** [comment]
Spec PRs are merged, I just need to add some tests and then this will be ready to go

File: common/eth2/src/lib.rs [hunk 1]
```diff
@@ -516,6 +516,29 @@ impl BeaconNodeHttpClient {
...
+            .push("states")
+            .push(&state_id.to_string())
+            .push("randao");
+
+        if let Some(epoch) = epoch {
+            path.query_pairs_mut()
+                .append_pair("epoch", &epoch.to_string());
+        }
+
+        self.get(path).await
```
Comment (realbigsean) [inline common/eth2/src/lib.rs]: Think this should probably use `get_opt` and return an optional on a 404

**Comment by realbigsean** [comment]
Added some tests here and the `get_opt` comment https://github.com/sigp/lighthouse/commit/e3bed81745bad0ed81af0324fb961812a0dc994e

The rest looks good to me!

**Comment by michaelsproul** [comment]
Thank you, that's a real timesaver!

**Review by realbigsean — APPROVED** [review]

**Pull request closed without merging**